src/pages/MembersPage/MembersActivation/Activation/Activation.jsx

```diff
@@ -71,7 +71,7 @@ function Activation() {
       <h3 className="text-base font-semibold">Member activation</h3>
       <p>
         <span className="text-lg font-semibold">{activatedUserCount}</span>{' '}
-        active members of{' '}
+        actived members of{' '}
```

Comment:
@jerrodcodecov typo here 👀 

Optional: if you want the true developer experience, I'd encourage you to change any mention of `activeMembers` to `activatedMembers`, so that it better represents the text you're fetching. 

Let me know if you need a hand with any of it